### PR TITLE
[Word] (document) Compare documents

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
In response to a request via verbatim

### CODE REVIEW

File: samples/word/99-preview-apis/compare-documents.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to compare two documents (the current one and a specified external one).</p>
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/compare-documents.yaml:42]:
```suggestion
            <p>This sample shows how to compare two documents: the current one and a specified external one.</p>
```

File: samples/word/99-preview-apis/compare-documents.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <label>File (with absolute path): </label>
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/compare-documents.yaml:47]: Could this be a file picker instead?

**Review by AlexJerabek — APPROVED** [review]
Approving, with the understanding that there will be another pass to check the comparison logic.

File: samples/word/99-preview-apis/compare-documents.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/compare-documents.yaml:47]: @AlexJerabek I'll look into that

Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/compare-documents.yaml:47]: Looks like, for security reasons, can't get absolute path from the browser. The filename is real but nothing else - you get C:\fakepath\Document.docx

**Pull request merged**